code/modules/client/preferences.dm

```diff
@@ -134,13 +134,31 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	var/auto_fit_viewport = TRUE
 
+	/// New TGUI Preference preview
+	var/map_name = "player_pref_map"
```

Comment:
doenst those need a name such as that changes per datum/preferences
"camera_console_[REF(src)]_map"

Reply:
no, since everything is inside clients vars